### DISCUSSION TITLE
Multilingual audio transcription - possible using whisper?

### DISCUSSION THREAD

**Discussion by anuragrawal2024** [openai/whisper#2014] [General]
Hi All,

I have a use case where my audios have both English and Spanish speakers and I want to transcribe them. Is it possible to get transcription with both languages using whisper? If not, are there any alternatives? Thanks!

**Comment by glangford** [comment]
* https://github.com/openai/whisper/discussions/2009

**Comment by toanhuynhnguyen** [comment]
For Multilingual audio transcription, you can try Microsoft Cognitive Service.
